scripts/install.sh

```diff
@@ -25,13 +39,14 @@ echo "\nWorking directory at ${tmpdir}\n"
 cd $tmpdir
```

Comment:
Nit: We could echo something like:
```
echo "Fetching ${VERSION} from https://github.com/${REPO}/tree/${BRANCH}"
```
Just to show any overrides

Reply:
done.